config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -3013,6 +3013,71 @@ presubmits:
           defaultMode: 256
           secretName: ssh-security
     trigger: (?m)^/test( | .* )pull-security-kubernetes-bazel-test,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    cluster: security
+    context: pull-security-kubernetes-conformance-image-test
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+    name: pull-security-kubernetes-conformance-image-test
+    optional: true
+    rerun_command: /test pull-security-kubernetes-conformance-image-test
+    spec:
+      containers:
+      - args:
+        - --ssh=/etc/ssh-security/ssh-security
+        - --job=$(JOB_NAME)
+        - --root=/go/src
+        - --repo=k8s.io/test-infra=master
+        - --repo=github.com/kubernetes-security/kubernetes=$(PULL_REFS)
+        - --repo=sigs.k8s.io/kind=master
+        - --service-account=/etc/service-account/service-account.json
+        - --upload=gs://kubernetes-security-prow/pr-logs
+        - --scenario=execute
+        - --
+        - bash
+        - --
+        - -c
+        - cd ./../../k8s.io/kubernetes && source ./../test-infra/experiment/kind-conformance-e2e.sh
+        env:
+        - name: PARALLEL
+          value: "true"
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-master
+        name: ""
+        resources:
+          requests:
+            cpu: "2"
+            memory: 9000Mi
+        securityContext:
+          privileged: true
+        volumeMounts:
```

Comment:
there's a preset for this now: `preset-kind-volume-mounts: "true"`

Reply:
This is a auto generated thing too!